confluence-xml/src/main/java/org/xwiki/contrib/confluence/filter/internal/input/ConfluenceConverterListener.java

```diff
@@ -45,10 +47,12 @@
 import org.xwiki.contrib.confluence.filter.input.ConfluenceInputContext;
 import org.xwiki.contrib.confluence.filter.input.ConfluenceProperties;
 import org.xwiki.contrib.confluence.filter.input.ConfluenceXMLPackage;
+import org.xwiki.contrib.confluence.filter.internal.ConfluenceFilter;
```

Comment:
I'm not sure you are really using this import but maybe I missed something.

Reply:
They are indeed unused. I expected the build to fail tho..

Reply:
Now when the file is excluded from checkstyle.